study.md

```diff
@@ -74,13 +76,14 @@ What are the similarities and differences between Ruby's and JavaScript's
 Similarities:
 
 ```md
-<!-- your answer here -->
+In both languages the itmes in the parantheses will be pushed to begining of the
+array
 ```
 
 Differences:
 
 ```md
-<!-- your answer here -->
+No fucnitonal differences.
```

Comment:
Ruby's unshift array method returns the modified array, allowing method chaining.

JavaScript's unshift array method returns the length of the modified array.